### PR TITLE
OSDOCS-248: Including EBS Persistent Storage.

### PR DESCRIPTION
**Pull request by huffmanca**
OSDOCS-248: This includes instructions for configuring AWS EBS persistent storage.

The current content is fairly minimal, and any suggestions or advice would be appreciated.

This is for 4.0.

### CODE REVIEW

**Comment by huffmanca** [comment]
@openshift/team-documentation Requesting peer review.

File: modules/storage-persistent-storage-aws-maximum-volumes.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage-aws.adoc
+
+[[maximum-number-of-ebs-volumes-on-a-node]]
+= Maximum Number of EBS Volumes on a Node
+
+By default, {product-title} supports a maximum of 39 EBS volumes attached to one
+node. This limit is consistent with the
+link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html#linux-specific-volume-limits[AWS
```
Comment (ahardin-rh) [inline modules/storage-persistent-storage-aws-maximum-volumes.adoc]: s/AWS Volume Limits/AWS volume limits

File: modules/storage-persistent-storage-aws-volume-format.adoc [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage-aws.adoc
+
+[[volume-format-aws]]
+= Volume Format
+Before {product-title} mounts the volume and passes it to a container, it checks
+that it contains a file system as specified by the `*fsType*` parameter in the
```
Comment (ahardin-rh) [inline modules/storage-persistent-storage-aws-volume-format.adoc]: s/`*fsType*`/`fsType`

File: modules/storage-persistent-storage-creating-volume-claim.adoc [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+.Procedure
+
+. In the {product-title} console, click *Storage* -> *Persistent Volume Claims*.
+
+. In the persistent volume claims overview, click *Create Persistent Volume
+Claim*.
+
+. Define the desired options on the page that appears.
+
+.. Select the storage class created previously from the drop down.
```
Comment (ahardin-rh) [inline modules/storage-persistent-storage-creating-volume-claim.adoc]: s/drop down/drop-down menu

**Comment by ahardin-rh** [comment]
@huffmanca Just a few minor suggestions from me :+1:

File: modules/storage-persistent-storage-aws-maximum-volumes.adoc (hunk 1 above)
Comment (huffmanca) [reply modules/storage-persistent-storage-aws-maximum-volumes.adoc]: Incorporated feedback.

File: modules/storage-persistent-storage-creating-volume-claim.adoc (hunk 3 above)
Comment (huffmanca) [reply modules/storage-persistent-storage-creating-volume-claim.adoc]: Incorporated feedback.

File: modules/storage-persistent-storage-aws-volume-format.adoc (hunk 2 above)
Comment (huffmanca) [reply modules/storage-persistent-storage-aws-volume-format.adoc]: Incorporated feedback.

**Comment by huffmanca** [comment]
@ahardin-rh, 

Thanks for the review! Feedback has been incorporated and pushed. Merging this PR.

**Pull request merged**